ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17-periodics.yaml

```diff
@@ -80148,7 +80148,17 @@ periodics:
     ci.openshift.io/generator: prowgen
     job-release: "4.17"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-gcp-ipi-disc-priv-amd-arm-day0-f28-destructive
+  name: periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-gcp-ipi-disc-priv-amd-mixarch-f28-longduration-cloud
+  reporter_config:
+    slack:
+      channel: '#forum-prow-long-duration-cloudinfra-testcases'
```

Comment:
We added this on purpose?

Reply:
"periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-gcp-ipi-disc-priv-amd-mixarch-f28-longduration-cloud" was not added by this PR, but I had to change the order of the tests. 

Before the PR:
```
[jiwei@jiwei-thinkpadt14gen4 release]$ git checkout master
Switched to branch 'master'
Your branch is up to date with 'origin/master'.
[jiwei@jiwei-thinkpadt14gen4 release]$ grep -E "name: periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-gcp-ipi-disc-priv-" ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17-periodics.yaml -n
80151:  name: periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-gcp-ipi-disc-priv-amd-arm-day0-f28
80239:  name: periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-gcp-ipi-disc-priv-amd-arm-day0-f28-destructive
80327:  name: periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-gcp-ipi-disc-priv-amd-mixarch-f28-longduration-cloud
[jiwei@jiwei-thinkpadt14gen4 release]$ 
```
After the PR:
```
[jiwei@jiwei-thinkpadt14gen4 release]$ git checkout gcp-multiarch-417
Switched to branch 'gcp-multiarch-417'
Your branch is up to date with 'origin/gcp-multiarch-417'.
[jiwei@jiwei-thinkpadt14gen4 release]$ grep -E "name: periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-gcp-ipi-disc-priv-" ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17-periodics.yaml -n
80063:  name: periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-gcp-ipi-disc-priv-amd-arm-day0-f28-destructive
80151:  name: periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-gcp-ipi-disc-priv-amd-mixarch-f28-longduration-cloud
80249:  name: periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-gcp-ipi-disc-priv-arm-amd-day0-f28
[jiwei@jiwei-thinkpadt14gen4 release]$

Reply:
get it.